src/messaging/tests/echo/BUILD.gn

```diff
@@ -19,13 +19,29 @@ import("${chip_root}/build/chip/tools.gni")
 
 assert(chip_build_tools)
 
-executable("chip-echo-requester") {
```

Comment:
I think this is the only option to validate TCP @pidarped

Reply:
I kept it below. It is just that `common.h/cpp` was re-used and I moved it into a source set.

Reply:
I have used echo protocol to validate TCP initially, although, currently, it can be done using the chip-tool and all-clusters-app interaction. However, I would recommend not removing the echo protocol code as it is immensely handy in quickly testing the lower parts of the stack